### PR TITLE
ceph-disk: retry on OSError

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
see also #14275

**Comment by tchaikov** [comment]
retest this please.

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-10-07_18:36:04-ceph-disk-wip-18162-kefu-distro-basic-vps/

the failed one is due to the missing 7.4 vps test nodes.

File: src/ceph-disk/ceph_disk/main.py [hunk 1]
```diff
@@ -721,6 +722,21 @@ def get_partition_mpath(dev, pnum):
         return None
 
 
+def retry(on_error=Exception, max_tries=10, wait=0.2, backoff=0):
```
Comment (dachary) [inline src/ceph-disk/ceph_disk/main.py]: much more elegant 💯

**Pull request merged**